backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/CStubsManager.kt

```diff
@@ -21,8 +21,8 @@ class CStubsManager(private val target: KonanTarget) {
         if (stubs.isEmpty()) return null
 
         val compilerOptions = mutableListOf<String>()
-        val sourceFileExtension = when (target.family) {
-            Family.OSX, Family.IOS -> {
+        val sourceFileExtension = when {
+            target.isAppleTarget -> {
```

Comment:
Guess previous version was readable enough.

Reply:
If we introduce `Family.TVOS`, `Family.WATCHOS` then it will be replaced with something like `target.family.isAppleFamily`/ `isDarwinBased`.

Reply:
TVOS and WATCHOS are pretty much the same as IOS, so why not just stick this way.